photon/license.md

```diff
@@ -0,0 +1 @@
+View [license information](https://github.com/vmware/photon/blob/master/LICENSE) for the software contained in this image.
```

Comment:
Bahahahaha `Sorry about that, but we can’t show files that are this big right now.`, that's epic.  :+1: